pkg/controllers/clusterresourceplacement/controller_test.go

```diff
@@ -2017,6 +2027,44 @@ func TestGetOrCreateClusterResourceSnapshot_failure(t *testing.T) {
 	}
 }
 
+func TestSplitSelectedResources(t *testing.T) {
```

Comment:
As discussed offline, the current UT is not good enough, we need to test different resource size combinations.